Tools/autotest/autotest.py

```diff
@@ -200,6 +200,21 @@ def run_step(step):
     if step == "prerequisites":
         return test_prerequisites()
 
+    if step == 'build.All':
```

Comment:
I'll have to ask you to please let this slide. Changing this means it becomes a tremendous amount of work to do changes on the later commits, since I get conflicts when cherry picking :cry: